packages/webpack/lib/configs/build.js

```diff
@@ -133,9 +135,21 @@ module.exports = function getConfig(config, name) {
           },
         }),
       ],
+      chunkIds: 'natural',
+
+      // TODO: re-consider setting this to false
```

Comment:
Is there an issue for this that we can track?

Reply:
Okay, this option replaces the use of the `ModuleConcatenationPlugin`-plugin. But when I remove the `ModuleConcatenationPlugin`-plugin and set this option to `true`, the issue is gone. Doesn't really make sense, but I'd go with that setting for now.